README.md

```diff
@@ -155,6 +173,11 @@ Logspout relies on the Docker API to retrieve container logs. A failure in the A
 * `SYSLOG_STRUCTURED_DATA` - datum for structured data field
 * `SYSLOG_TAG` - datum for tag field (default `{{.ContainerName}}+route.Options["append_tag"]`)
 * `SYSLOG_TIMESTAMP` - datum for timestamp field (default `{{.Timestamp}}`)
+* `MULTILINE_ENABLE_DEFAULT` - enable multiline logging for all containers (default `true`)
+* `MULTILINE_MATCH` - determines which lines the pattern should match, one of first|last|nonfirst|nonlast (default `nonfirst`)
```

Comment:
I'd love some explanation to what exactly thos eoptions mean:
e.g.
- first: match first line only and append following messages until you match the next one
- last: concatenate all messages until the pattern matches the next line
- nonlast: concatenate all matching lines, stop on first non-matching line, append it and start over
- nonfirst: vice versa

Reply:
@masterada would you mind adding docs around these options?

Reply:
@michaelshobbs if you're referring to first|last|nonfirst|nonlast, the explanations can be found in lines 151 to 155 :)

Reply:
ah i totally missed that. thanks!